src/pages/solutions/for-gis-users.astro

```diff
@@ -92,26 +92,32 @@ const page: IStandardPageMetadata = {
       UGRC's address locators are a quick and simple way to convert tabular address data into geospatial data that can
       be mapped! Our address locator is one of the most popular products we offer. Depending on your needs, there are a
       variety of ways to interact with it including through the UGRC API, an installable desktop application (macOS and
-      Windows), or through the locator tab in ArcGIS Pro. Our locator accepts a variety of input values including civic
-      addresses and latitude and longitude coordinates. Depending on the locator style you choose, users can pass in a
-      single address or a list of addresses. You can learn more, including how to access them, on our address locator
-      page.
+      Windows), or through the locator tab in ArcGIS Pro. Our locator accepts a variety of input values including common
+      addresses or latitude and longitude coordinates. Depending on the locator style you choose, users can pass in a
+      single address or a list of addresses. You can learn more, including how to access our locators, on our address
+      locator page.
```

Comment:
do we have a locator page?

Reply:
good call. fixed to refer to geocoders and the address data pages.